code_challenges/code_wars/rot13.py

```diff
@@ -1,18 +1,23 @@
 def rot13(a_string):
     code = ""
+    amount_to_shift = 13
+    upper_alpha_min = 65
+    upper_alpha_max = 90
+    lower_alpha_min = 97
```

Comment:
The only times you use the mins you add one to them. It would be better to include that in the variable to start with.